lib/eclipse/share/keywords/000_Eclipse100/R/RTEMP

```diff
@@ -0,0 +1,4 @@
+{"name" : "RTEMP" , "sections" : ["PROPS","SOLUTION"],
+ "items" :
+ [{"name" : "TEMP" , "value_type" : "DOUBLE" , "default" : 15.555 , "dimension" : "Temperature"}]}
```

Comment:
actually, the 15.5 °C are right. my confusion was caused by a possible bug of the 'units' command line utility: it produces 33.33 degC if asked for 60 degF...